app/views/brexit_campaign/_all_taxons.html.erb

```diff
@@ -0,0 +1,24 @@
+<section>
+  <% if @other_taxons.any? %>
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <%= render partial: 'featured_taxons', locals: { display_full_width: false } %>
```

Comment:
Do we need to pass `display_full_width`? Would it be possible to check `@other_taxons.any?` inside `featured_taxons.html.erb` instead?

Reply:
It would be possible to check this, you just then have implicit knowledge about instance variables a few layers deep. I'm happy to make this change if we want to.

Reply:
I'm easy either way 👍